### PR TITLE
[Compute] vm/vmss update: Add --priority; vm/vmss update: Add --max-price

### PR DESCRIPTION
**Pull request by qwordy**
Issue #11187 

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: src/azure-cli/azure/cli/command_modules/vm/_params.py [hunk 1]
```diff
@@ -749,6 +740,11 @@ def load_arguments(self, _):
                           "To enable Multitenant Hosting Rights for Windows 10, use 'Windows_Client'. " \
                           "For more information see the Azure Windows VM online docs."
             c.argument('license_type', help=license_msg, arg_type=get_enum_type(['Windows_Server', 'Windows_Client', 'None']))
+            c.argument('priority', resource_type=ResourceType.MGMT_COMPUTE, min_api='2019-03-01',
```
Comment (MyronFanQiu) [inline src/azure-cli/azure/cli/command_modules/vm/_params.py]: curious about the `min_api`. If the min_api is different in vm and vmss, should we keep them separated?

Comment (qwordy) [reply src/azure-cli/azure/cli/command_modules/vm/_params.py]: Yes, we should. This the cause of error in Profile 2019-03-01.

File: src/azure-cli/azure/cli/command_modules/vm/_params.py [hunk 2]
```diff
@@ -746,6 +746,11 @@ def load_arguments(self, _):
             c.argument('max_price', min_api='2019-03-01', type=float, is_preview=True,
                        help='The maximum price (in US Dollars) you are willing to pay for a low priority VM/VMSS. -1 indicates that the low priority VM/VMSS should not be evicted for price reasons')
 
+    with self.argument_context('vmss create') as c:
```
Comment (yungezz) [inline src/azure-cli/azure/cli/command_modules/vm/_params.py]: where's max-price as the PR title mentioned?

**Pull request merged**